src/components/molecules/content/portable-text/ListItems.svelte

```
@@ -0,0 +1,46 @@
+{#if value && children && children.length > 0}
+	{#if listItem === 'number'}
+		<ol class="list-decimal pl-5">
+			{#each children as child}
+				<li class="text-text-main">
+					{#if child.marks?.includes('strong')}
+						<strong>{child.text || ''}</strong>
+					{:else}
+						{child.text || ''}
+					{/if}
+				</li>
+			{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)